tests/v2/validation/provisioning/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	CNIs               []string                 `json:"cni" yaml:"cni"`
 	Providers          []string                 `json:"providers" yaml:"providers"`
 	NodeProviders      []string                 `json:"nodeProviders" yaml:"nodeProviders"`
+	Hardened           bool                     `json:"hardened" yaml:"hardened"`
```

Comment:
Quick question Markus, if Hardened isn't included in the config file at all, does it default to false?

Reply:
Correct, I just double-checked to confirm. I removed from the config file and the cluster was not hardened at all.

Reply:
Awesome! Thank you!